recipes/ptxcompiler/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ptxcompiler" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/ptxcompiler/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c7021a0946478404c4055a612fd725af95d53ec23eb3dd8d8bc0b5ac55eb44d0
+  fn: ptxcompiler.tar.gz
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  skip: true  # [cuda_compiler_version == "None"]
+  skip: true  # [cuda_compiler_version == "10.2"]
+  skip: true  # [cuda_compiler_version == "11.0"]
+  script_env:
+    - CPATH=/usr/local/cuda/include
```

Comment:
Have you tried removing these definitions?

Reply:
Yes. Build fails with: [cannot find -lnvptxcompiler_static](https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/470564/logs/61)
I also tried using CUDA_HOME (as in https://github.com/conda-forge/rmm-feedstock/blob/main/recipe/meta.yaml) instead of CPATH and LIBRARY_PATH.
I'll restore these env vars now.